### DISCUSSION TITLE
how to customize the log in screen.

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#1302] [Help] [answered]
Hi @joebudi  I can not figure out how to customize the login screen with a custom logo. 
also i can only get a web link photo to work for the logo in the upper left hand corner. how can I add picture from a local file?

**Comment by savannt** [comment, marked as answer]
It is not possible to change the login screen unless you self host and edit the source code yourself.

Though you can utilize the Budibase Public API to create a form that completes the login request & saves the login cookies
Or, host a login/register page on a non-budibase site and redirect under the same domain

https://docs.budibase.com/reference/post_users
Template: https://github.com/Budibase/budibase/discussions/6098
